compression-service/EXML-converter-service/src/main/java/io/javabrains/EXMLconverterservice/resources/XMLConverter.java

```diff
@@ -0,0 +1,66 @@
+package io.javabrains.EXMLconverterservice.resources;
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import com.agiledelta.efx.EFXFactory;
+import com.agiledelta.efx.text.Transcoder;
+import com.agiledelta.efx.EFXException;
+
+
+public class XMLConverter {
+
+    public byte [] compressXML(String xml) throws EFXException, IOException {
+
+            byte[] exmlByteResult = null;
+            try{
+            EFXFactory factory = EFXFactory.newInstance();
+            File schemaFile = new File("GCSS-XSD/RICE-Master-Schema.xsd");
+            factory.setSchema(schemaFile);
+            ByteArrayOutputStream exmlStream = new ByteArrayOutputStream();
+            InputStream xmlStream = new ByteArrayInputStream(xml.getBytes());
+            Transcoder transcoder = factory.newTranscoder();
```

Comment:
consider making the factory, exmlStream, and transcoder static class properties to avoid instantiating on every method call